connect/src/main/java/com/stripe/android/connect/webview/serialization/SetCollectMobileFinancialConnectionsResultPayloadJs.kt

```diff
@@ -0,0 +1,63 @@
+package com.stripe.android.connect.webview.serialization
+
+import com.stripe.android.core.StripeError
+import com.stripe.android.core.exception.StripeException
+import com.stripe.android.core.model.serializers.StripeErrorSerializer
+import com.stripe.android.financialconnections.FinancialConnectionsSheetResult
+import com.stripe.android.financialconnections.model.FinancialConnectionsAccount
+import com.stripe.android.model.Token
+import com.stripe.android.model.parsers.TokenSerializer
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal data class SetCollectMobileFinancialConnectionsResultPayloadJs(
+    val id: String,
+    val financialConnectionsSession: FinancialConnectionsSessionJs?,
+    @Serializable(TokenSerializer::class) val token: Token?,
+    @Serializable(StripeErrorSerializer::class) val error: StripeError?,
```

Comment:
👀 `@Serializable` here because we're parsing them using `ModelJsonParser` not using kotlinx.serialization.